service_info/static/less/site.less

```diff
@@ -1,3 +0,0 @@
-/*
-*   Site Styles
-*/
```

Comment:
This empty file was added because of this ticket: https://github.com/caktus/django-project-template/issues/136

I don't know for sure if that's still an issue, though... I guess we'll see on deploy?